<?xml version="1.0" ?>

<launch>    
    
    <param name="gazebo"                   value="false"    type="bool"/>
    <param name="gazebo_robot_name"        value="robotis_humanoid"/>
    
    <param name="offset_table"             value="$(find op3_manager)/config/offset.yaml"/>
    <param name="robot_file_path"          value="$(find op3_manager)/config/OP3.robot"/>
    <param name="init_file_path"           value="$(find op3_manager)/config/dxl_init_OP3.yaml"/>
    
    <!-- THORMANG3 Manager -->
    <node name="op3_walking_tuner_node" pkg="op3_walking_tuner" type="op3_walking_tuner_node" output="screen"/>
    
</launch>
